etsin_finder/frontend/js/components/qvain/participants/index.jsx

```diff
@@ -477,80 +490,6 @@ const Column = styled.div`
   width: 50%;
 `
 
```

Comment:
This is where I removed all the styled components to put them to their own file.